### ISSUE TITLE
tt-wdio formFactor testing inconsistencies

### ISSUE THREAD

**Issue by ShettyAkarsh** [cerner/terra-toolkit-boneyard#61]
Need to investigate why tt-wdio scripts passes all the tests for the form factors that are passed to the wdio:test:local command together. but if the same script is called for individual form factor like 'Form_Factor='small' npm run wdio' the test fails with screen shot mismatch.

Expectation is that the test should fail when there is a mismatch in the screenshots even though if we pass the form factors together.

**Comment by mjhenkes** [comment]
@ShettyAkarsh do you have an example test where you're seeing this problem? is it all tests?

**Comment by emilyrohrbough** [comment]
@ShettyAkarsh  You need to pass form factor like `FORM_FACTOR` not `Form_Factor`

**Comment by ShettyAkarsh** [comment]
This issue is not consistent mainly seen when we run wdio for the js component using 'npm run wdio:local' not seen for terra monorepo's though. usually consumers are seeing tests getting passed in local but in jenkins it starts failing for random form_factors. 
so in local when they try to pass the viewport value (which is failing in jenkins) to FORM_FACTOR param along with their wdio command they see the same issue which they are getting in jenkins.
